arch/arm/configs/bcm2709_defconfig

```diff
@@ -570,6 +570,7 @@ CONFIG_INPUT_TOUCHSCREEN=y
 CONFIG_TOUCHSCREEN_ADS7846=m
 CONFIG_TOUCHSCREEN_EGALAX=m
 CONFIG_TOUCHSCREEN_EKTF2127=m
+CONFIG_TOUCHSCREEN_GOODIX=m
```

Comment:
I can see why you put the new addition here, but the Kconfig file entries aren't in alphabetical order. If you use the `savedefconfig` make target you'll find that Kconfig puts it before the EKTF2127 (which is mysteriously absent from bcmrpi_defconfig - not your problem). Can you move it?